tony-core/src/test/java/com/linkedin/tony/TestTonyConfigurationFields.java

```diff
@@ -32,6 +32,9 @@ public void initializeMemberVariables() {
     xmlPropsToSkipCompare.add(TonyConfigurationKeys.getGPUsKey(Constants.WORKER_JOB_NAME));
     xmlPropsToSkipCompare.add(TonyConfigurationKeys.getResourcesKey(Constants.WORKER_JOB_NAME));
     configurationPropsToSkipCompare.add(TonyConfigurationKeys.DOCKER_IMAGE);
+    // These properties are in TonyConfigurationKeys, but not in tony-default.xml
+    // so we skip this check
+    configurationPropsToSkipCompare.add(TonyConfigurationKeys.TONY_HISTORY_LOCATION);
```

Comment:
Why do you want to skip checking this config? I think it's informative to have a default value present in `tony-default.xml`.

Reply:
Discussed offline. We should not skip this check as all the configurations in tony-site.xml will also be in tony-default.xml.